client/scss/components/_responsive-defaults.scss

```diff
@@ -0,0 +1,3 @@
+img {
```

Comment:
Are we 1000% sure we want all images everywhere to have this?
Not a class of `.responsive-image`?

What about the author image, which never changes size?